server/stream/streamBroadcast.js

```diff
@@ -165,6 +165,11 @@ function startStreamCastBroadcast(value) {
 		_dontPrintErrors: LoggerManager.logLevel < 2,
 	});
 
+	// Ignore collection events to prevent store in memory
+	// necessary to prevent memory lead due to the usage
+	// of DDP subscribe
+	connection._livedata_data = () => {};
```

Comment:
doing this actually breaks streamcast, all events are ignored and communication doesn't happen.

another way of preventing this leak to happen, is to actually register a store for the `broadcast-stream` collection.

so I changed the current line 180 (`connection._stream.on('message', function(raw_msg) {` for the following code:

```javascript
	connection.registerStore('broadcast-stream', {
		update({ fields }) {
			const { streamName, eventName, args } = fields;

			if (!streamName || !eventName || !args) {
				return;
			}

			if (connection.broadcastAuth !== true) {
				return 'not-authorized';
			}

			const instance = StreamerCentral.instances[streamName];
			if (!instance) {
				return 'stream-not-exists';
			}

			if (instance.serverOnly) {
				return instance.__emit(eventName, ...args);
			}
			return instance._emit(eventName, args);
		},
	});
```

looks like even a less hacky solution, wdyt?